### DISCUSSION TITLE
Is placing equations in LaTeX table caption supported?

### DISCUSSION THREAD

**Discussion by nclJoshCowley** [quarto-dev/quarto-cli#6024] [Q&A] [answered]
### Description

Asking this as a question before filing an issue as unsure if this is a pandoc issue.

Table captions seem to be losing the equation environment at some point.

Consider the following .qmd,

````
---
title: "tbl-cap-eq"
format: pdf
keep-tex: true
---

```{r tbl-label}
#| echo: false
#| tbl-cap: |
#|   Text before equation, $x = 1, \dots, 3$.
gt::gt(head(mtcars, 3))
```

```{r fig-label}
#| echo: false
#| warning: false
#| fig-cap: |
#|   Text before equation, $x = 1, \dots, 3$.
ggplot2::qplot(1:3)
```
````

Expectation is that both will keep the equation when converted to LaTeX. Not the case with the table; is the case with the figure.

The LaTeX output, for the captions only, for the table is

```
\caption{\label{tbl-label}Text before equation, x = 1, \textbackslash dots, 3. }\tabularnewline
```

and for the figure,

```
\caption{\label{fig-label}Text before equation, \(x = 1, \dots, 3\).}
```

where the `$` is swapped out for `\(` in the figures case only.

Can others reproduce this? (I'm using Quarto v1.2.313)

Should I file a git issue for this?

**Comment by baptiste** [comment, marked as answer]
You may want to try a more recent Quarto; I don't see a problem with v1.3.353.

**Reply by cderv** [reply]
Yes I believe this was fixed in 1.3 as before `*-cap` were not processed as Markdown content. So I confirm that updating will solve this.
